frontend/lib/profile/view/profile.view.dart

```diff
@@ -97,9 +97,12 @@ class _ProfileState extends State<Profile> {
       padding: EdgeInsets.all(16.0),
       child: Column(
         children: [
-          CircleAvatar(
-            radius: 50,
-            backgroundImage: AssetImage(profileImage),
+          Hero(
```

Comment:
I might be missing something but I don't see a Hero animation for the profile picture. Is there a reason why you're using the `Hero` widget here?